x-pack/plugins/security_solution/cypress/e2e/detection_alerts/expandable_flyout/alert_details_left_panel_session_view_tab.cy.ts

```diff
@@ -19,7 +19,7 @@ import { waitForAlertsToPopulate } from '../../../tasks/create_new_rule';
 
 // Skipping these for now as the feature is protected behind a feature flag set to false by default
 // To run the tests locally, add 'securityFlyoutEnabled' in the Cypress config.ts here https://github.com/elastic/kibana/blob/main/x-pack/test/security_solution_cypress/config.ts#L50
-describe.skip(
+describe(
```

Comment:
you should keep the skip here. I think @PhilippeOberti  is leaving these skipped since the feature flag isn't default enabled

Reply:
Done.